packages/api/src/middleware/acl/checkJwt.js

```diff
@@ -26,6 +26,7 @@ const checkJwt = jwt({
     '/user/accept_invitation',
```

Comment:
I can't leave a comment on the top of this file for some reason, but can you update the copyright in this file to: 
```javascript
/*
 *  Copyright 2019, 2020, 2021, 2022 LiteFarm.org
 *  This file is part of LiteFarm.
 *
 *  LiteFarm is free software: you can redistribute it and/or modify
 *  it under the terms of the GNU General Public License as published by
 *  the Free Software Foundation, either version 3 of the License, or
 *  (at your option) any later version.
 *
 *  LiteFarm is distributed in the hope that it will be useful,
 *  but WITHOUT ANY WARRANTY; without even the implied warranty of
 *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
 *  GNU General Public License for more details, see <https://www.gnu.org/licenses/>.
 */
 ```